hash/risc0/bench/src/main.rs

```diff
@@ -0,0 +1,88 @@
+mod benches;
+use benches::{
+    sha256::sha_bench,
+    keccak::keccak_bench,
+    blake2b::blake2b_bench,
+    blake3::blake3_bench,
+    poseidon2_bn128::poseidon2_bn128_bench,
+    poseidon2_babybear::poseidon2_babybear_bench,
+    poseidon2_babybear_native::poseidon2_babybear_native_bench,
+};
+use rand::Rng;
+use std::process;
+
+fn generate_bytes(size: usize) -> Vec<u8> {
+    let mut rng = rand::thread_rng();
+    (0..size).map(|_| rng.gen()).collect()
+}
+
+
+
+fn main() {
+    let args: Vec<String> = std::env::args().collect();
+
+    if args.len() != 3 {
+        println!("Wrong number of arguments");
+        // Exit the program with a non-zero exit code
```

Comment:
It would be nice to print out a help text here for the user what kind of arguments are available. I find this a good practice in general.